dhis-2/dhis-services/dhis-service-node/src/main/java/org/hisp/dhis/node/types/SimpleNode.java

```diff
@@ -33,58 +33,83 @@
 import org.hisp.dhis.node.NodeType;
 import org.hisp.dhis.node.exception.InvalidTypeException;
 
+import java.util.Objects;
+
 /**
  * @author Morten Olav Hansen <mortenoh@gmail.com>
  */
-public class SimpleNode extends AbstractNode
+public class SimpleNode
+    extends AbstractNode
 {
-    /**
-     * Value of this node.
-     */
-    private final Object value;
-
-    /**
-     * Is this node considered a attribute.
-     */
-    private boolean attribute;
-
-    public SimpleNode( String name, Object value )
-    {
-        super( name, NodeType.SIMPLE );
-        this.value = value;
-        this.attribute = false;
-    }
-
-    public SimpleNode( String name, Object value, boolean attribute )
-    {
-        this( name, value );
-        this.attribute = attribute;
-    }
-
-    public Object getValue()
-    {
-        return value;
-    }
-
-    public boolean isAttribute()
-    {
-        return attribute;
-    }
-
-    public void setAttribute( boolean attribute )
-    {
-        this.attribute = attribute;
-    }
-
-    @Override
-    public <T extends Node> T addChild( T child )
-    {
-        throw new InvalidTypeException();
-    }
-
-    @Override
-    public <T extends Node> void addChildren( Iterable<T> children )
-    {
-        throw new InvalidTypeException();
-    }
+        /**
+         * Value of this node.
+         */
+        private final Object value;
+
+        /**
+         * Is this node considered a attribute.
+         */
+        private boolean attribute;
+
+        public SimpleNode( String name, Object value )
+        {
+                super( name, NodeType.SIMPLE );
+                this.value = value;
+                this.attribute = false;
+        }
+
+        public SimpleNode( String name, Object value, boolean attribute )
+        {
+                this( name, value );
+                this.attribute = attribute;
+        }
+
+        public Object getValue()
+        {
+                return value;
+        }
+
+        public boolean isAttribute()
+        {
+                return attribute;
+        }
+
+        public void setAttribute( boolean attribute )
+        {
+                this.attribute = attribute;
+        }
+
+        @Override
+        public <T extends Node> T addChild( T child )
+        {
+                throw new InvalidTypeException();
+        }
+
+        @Override
+        public <T extends Node> void addChildren( Iterable<T> children )
+        {
+                throw new InvalidTypeException();
+        }
+
+        @Override
+        public boolean equals( Object o )
+        {
+                if ( this == o )
+                        return true;
```

Comment:
Add brackets { } for single lines according to code style.

Also use indendation = 4 spaces.